gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Table.java

```diff
@@ -322,14 +322,15 @@ public Builder toBuilder() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof Table
+  public final boolean equals(Object obj) {
+    return this == obj
+        || obj instanceof Table
```

Comment:
Fine as is (as nothing extends it) however wouldn't it be better if used 'obj != null && object.getClass().equals(Table.class)` consistently?